src/modules/esl-utils/dom/events/target.ts

```diff
@@ -48,7 +60,7 @@ export class SyntheticEventTarget implements EventTarget {
     Object.defineProperty(e, 'currentTarget', targetDescriptor);
     Object.defineProperty(e, 'srcElement', targetDescriptor);
 
-    this._listeners[e.type]?.forEach((listener) => {
+    this.getEventListeners(e.type).forEach((listener) => {
```

Comment:
Then we should make a check for e.type
Because if for some reason we pass an event with an empty type we will get totally incorrect dispatching behavior